.github/workflows/unit-test-widget.yml

```diff
@@ -0,0 +1,51 @@
+name: Generic widget unit test
+
+on:
+  workflow_call:
+    inputs:
+      test_command:
+        description: "Command to run widget unit tests. For example: yarn test"
+        required: true
+        type: string
+      skip_run:
```

Comment:
add a skip file for the workflow to run when files that are not in the path provided in the ci yaml file are modified.